tox.ini

```diff
@@ -4,15 +4,16 @@ envlist =
     style
     packaging
     py{36,37}-dj{22,31,32}-sqlite
-    py{38,39}-dj{22,31,32,main}-sqlite
+    py{38,39}-dj{22,31,32,40,main}-sqlite
     py{36,37,38,39}-dj{22,31,32}-{postgresql,mysql}
-    py{310}-dj{32,main}-{sqlite,postgresql,mysql}
+    py{310}-dj{32,40,main}-{sqlite,postgresql,mysql}
```

Comment:
`py{38,39}-{40,main}-{postgresql,mysql}` is missing?

Reply:
Ah yes, thanks!

I'm *so* confused by this matrix 😅

Reply:
True, maybe:
```
    py{36,37}-dj{22,31,32}-{sqlite,postgresql,mysql}
    py{38,39}-dj{22,31,32,40,main}-{sqlite,postgresql,mysql}
    py{310}-dj{32,40,main}-{sqlite,postgresql,mysql}
```
it's more readable to me :thinking:

Reply:
Django 3.2 doesn't officially support Python 3.10.

Reply:
Very helpful, thank you!